src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static List<User> users;
+    private final RegistrationServiceImpl util = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void setUp() {
+        users = new ArrayList<>();
+        users.add(new User("SuperAdam", "123456", 19));
+        users.add(new User("SuperAdam", "randomPassword", 99));
```

Comment:
The second user in the `users` list has the same login as the first user. This is intentional for testing non-unique login, but ensure that other test cases do not rely on this user unless intended.